backend/tests/mw_drw_proxy/README.md

```diff
@@ -0,0 +1,16 @@
+# MiniWallet <=> Reference Wallet Proxy
+
+This will run a proxy which allows the DMW test suite to communicate with DRW.
+
+For example, when the DRW requests to set up a user, the proxy registers a user, updates their information with KYC, 
+and deposits any initial test balances that the DMW required, before returning an identifier for that user which DMW 
+will reuse in future calls.
+
+
+### Usage:
+(from inside the venv) 
+```
+DRW_URL_PREFIX=http://reference_wallet:8080 MW_DRW_PROXY_PORT=3130 python3 tests/mw_drw_proxy/proxy.py
+```
+This will launch the proxy server on `http://localhost:3130`, and will proxy the requests to an instance of 
+DRW running at `http://reference_wallet:8080`
```

Comment:
we probably need add a make task to run tests, something like `make dmw-test`, which runs the test like what we did for e2e tests, so that we can add these test into build, but can be added next PR.

Reply:
yeah I think being able to perform the task will require a docker image of the python SDK

Reply:
why require a docker image of the python sdk?
current DRW depends on python sdk, we just need to include all dependencies: use 'diem[all]'

Reply:
is it the only way we can connect DRW instance in docker to the DMW instance by using docker and create bridge between them?

Reply:
sorry I just realized we're actually pushing the python sdk to pypi- we can absolutely use that, yeah. Lets put that in a follow up PR as I'm not sure what the pypi publishing process is, but the package there isn't up to date (I'd like to be able to update the package to desired version before adding to makefile so I can test)

Reply:
sounds good.

Reply:
Documentation added to readme :-)